game-core/src/main/java/games/strategy/triplea/delegate/AbstractDelegate.java

```diff
@@ -44,7 +44,7 @@ public void setDelegateBridgeAndPlayer(final IDelegateBridge delegateBridge) {
    */
   @Override
   public void start() {
-    // nothing to do here
+    BattleCalculator.clearOolCache();
```

Comment:
Curious why the cache clear here in addition to step changed?
I think the `#start` might have been pretty close to being a no-op everywhere and possibly removed. Is this a safe-measure or necessary?